optics-core/src/Optics/Fold.hs

```diff
@@ -76,7 +76,7 @@ module Optics.Fold
   , backwards_
 
   -- * Monoid structures
-  -- | 'Fold' admits (at least) two monoid structures:
+  -- | 'Fold' admits (at least) two monoid structures: #monoids#
```

Comment:
what's that `#monoids#` is?

Reply:
Haddock syntax for an anchor, so I can link to this section from elsewhere via `"Optics.Fold#monoids"`.

Reply:
TIL